web/client/components/data/query/GeometryDetails.jsx

```diff
@@ -144,12 +144,20 @@ class GeometryDetails extends React.Component {
                 center.x - this.circle.radius, center.y - this.circle.radius,
                 center.x + this.circle.radius, center.y + this.circle.radius
             ];
-
+            let coordinates = CoordinatesUtils.calculateCircleCoordinates(center, this.circle.radius, 100);
+            if (this.props.geometry && this.props.geometry.projection === "EPSG:4326") {
```

Comment:
Can't we reproject from any projection that is not the current map projection to the current map projection?

Reply:
I could try, but don't know if it works, because of #2222